contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcMeasureConstants.java

```diff
@@ -160,33 +206,114 @@
    *
    * @since 0.8
    */
+  @Deprecated
   public static final MeasureLong RPC_CLIENT_RESPONSE_COUNT =
       Measure.MeasureLong.create(
           "grpc.io/client/response_count", "Number of client RPC response messages", COUNT);
 
+  /**
+   * {@link Measure} for total bytes sent across all request messages per RPC.
+   *
+   * @since 0.13
+   */
+  public static final MeasureDouble RPC_CLIENT_SENT_BYTES_PER_RPC =
```

Comment:
Same here. Why not GRPC_CLIENT_SENT_BYTES_PER_RPC?

Reply:
Done.